src/users/users.service.ts

```diff
@@ -1,15 +1,30 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, ConfigService } from '@nestjs/common';
 import { CreateUserDto } from './dto/create-user.dto';
 import { UpdateUserDto } from './dto/update-user.dto';
 import { Prisma } from '../database/prisma.service';
+import Stripe from 'stripe';
 
 @Injectable()
 export class UsersService {
-  constructor(private prisma: Prisma) {}
+  constructor(private prisma: Prisma, private configService: ConfigService) {
+    const stripeApiKey = this.configService.get<string>('STRIPE_SECRET_KEY');
+    if (!stripeApiKey) {
+      throw new Error('Stripe API key not configured.');
+    }
+    this.stripe = new Stripe(stripeApiKey, { apiVersion: '2020-08-27' });
+  }
 
   async create(createUserDto: CreateUserDto) {
+    const customer = await this.stripe.customers.create({
+      email: createUserDto.email,
+      name: createUserDto.name,
```

Comment:
need to send a meta data for stripe with the id of the user from user table. 
 @FumeDev